src/main/java/org/reaktivity/command/log/internal/LogCommand.java

```diff
@@ -54,6 +55,7 @@ public static void main(String[] args) throws Exception
             String directory = cmdline.getOptionValue("directory");
             boolean verbose = cmdline.hasOption("verbose");
             String type = cmdline.getOptionValue("type", "streams");
+            int interval = Integer.parseInt(cmdline.getOptionValue("interval", "0"));
```

Comment:
Instead of implementing the interval loop in each command, implement it here.  Store the Commands so that they can be recalled in the interval.